W 11673436 usermanagement assignprocess se (#108)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc

. Repeat the previous step until the user has all needed privileges.
. Click *Save*.

== Change Process Assignments of a User

> Instead of editing a project and adding a user to its project team, change several project assignments of a user at once:

Reviewer: "change several..." > "you can change several..."

Since it is an either-or situation, this is something they don't have to do. The "you can" reflects that in this context.

. Open the *Users* view of the *User Management* module.
. Click the *Assign Project* icon image:assignproject-icon.png[process-diagram symbol,1.5%,1.5%] in the panel of the user whose project assignments you want to change.

The view *Assign Process* shows you a list of all processes where the user is part of the process team.

> Change permissions for assigned projects to adapt to changed responsibilities:

Reviewer: ```suggestion
Change permissions for assigned projects to reflect changed user responsibilities:
```

> Change permissions for assigned projects to adapt to changed responsibilities:

Reviewer: Suggestion: To change permissions...

. In the view *Assign Process*, for each process, select the checkboxes to permit the phases to the user.
. Click *Save*.

> You cannot cannot change permissions for the project manager, who always has permissions for every phase.

Reviewer: ```suggestion
You cannot change permissions for the project manager, who always has permissions for every phase.
```

> You cannot cannot change permissions for the project manager, who always has permissions for every phase.

Reviewer 2: "cannot" is written twice.

> The table does not show advanced permissions, which are sometimes needed to transfer a process to the next phase. To change these, xref:processautomation-manage.adoc#edit-an-automation-project[edit the process data sheet] in the *Process Management* view of the *Process Automation* module.

Reviewer 2: I'd say "To change these permissions", edit...

> Assign additional processes to a user to support the process team:

Reviewer: Suggestion: to assign...

. In the view *Assign Process*, select a process from the dropdown *Select Process*.
. In the table *Process List*, in the row of the added process, select the checkboxes of all phases the user needs to work in.
. Click *Save*.

> Remove a user from a process team:

Reviewer: Suggestion: To remove...

> . In the view *Assign process*, click the the *Delete* icon image:delete-icon.png[trash symbol,1.5%,1.5%] for each process you want to remove from the user.

Reviewer: ```suggestion
. In the view *Assign process*, click the *Delete* icon image:delete-icon.png[trash symbol,1.5%,1.5%] for each process you want to remove from the user.
```

> . In the view *Assign process*, click the the *Delete* icon image:delete-icon.png[trash symbol,1.5%,1.5%] for each process you want to remove from the user.

Reviewer 2: "the" is written twice.

. Click *Enable Two-Factor Authentication*.
. Log in using your password and 2FA.

== Edit a User

> Edit a user to change his data or permissions.

Reviewer: We don't use gender-specific words such as "his." Make the text plural to get around this.

. In the *Users* view of the *User Management* module, click the icon *Edit* in the table row of the user you want to edit.
. Edit the form *Edit a User*.
+

> The properties are explained in the topic <<form-create-user, *Create a new User*>>.

Reviewer: For an explanation of the properties, see...
